### ISSUE TITLE
Please give an example in Readme of how it works

### ISSUE THREAD

**Issue by arielgabizon** [grondilu/bitcoin-bash-tools#11]

**Comment by RayMetz100** [comment]
+1  Tried these
chmod u+x bitcoin.sh
./bitcoin.sh
./bitcoin.sh --help
./bitcoin.sh newBitcoinKey
bash bitcoin.sh
bash bitcoin.sh --help
bash bitcoin.sh newBitcoinKey

none of these worked.  no error.  I think I need to add some code to the bottom that calls the methods above.  I don't think there is code that accepts command line input yet.  I will try and learn some more bash.

**Comment by RayMetz100** [comment]
I figured it out.  Add newBitcoinKey which is the name of the function you want to run to the end of the script.  I used vi or vim and pasted in the whole script, then added newBitcoinKey with no echo and no () at the end.  just newBitcoinKey on it's own line.  Then save and run ./bitcoin.sh and it will give you this output:

% bash bitcoin.sh   (or ./bitcoin.sh)
---
secret exponent:          0x75E574330554C4BBE46F00A998777C8AF279E15C3C9F03FB075CF376E82ACCB4
public key:
    X:                    CB29B09B09B8DB77D30973734D3898D884D83BC615A7A0BCC7624F4FF8633EA8
    Y:                    3DB43A02103C894DABE620316949125DBFE81BB611AA45229B05BEF8DC5C3AF0
compressed:
    WIF:                  L1AtLPbAYTfQP7rEfJ6KUYeVWU2Fbv2YM5WtzZNhNT8E8zuEMzaE
    bitcoin address:      15VgoXUuA2mUNcuzCZok6VFrzCxxi1C8EJ
uncompressed:
    WIF:                  5JiD5r57hCw1mS7dB34UJ7iaCD1jjMAs9dNGrk8dneBhb3f2BD7
    bitcoin address:      1BfYfnQqSYyfvRSygdsnGTv9Lfuk7PgHsT


I haven't tried the other functions yet.  I don't know how to pass switches in to call different functions without modifying the whole command each time, but that should be easy.  See http://ryanstutorials.net/bash-scripting-tutorial/bash-functions.php
